t/Email/set/create/only-one-part-in-html-body.t

```diff
@@ -0,0 +1,58 @@
+use jmaptest;
+
+use Data::GUID qw(guid_string);
+
+test {
+  my ($self) = @_;
+
+  my $account = $self->any_account;
+  my $tester  = $account->tester;
+
+  my $mbox = $account->create_mailbox;
+
+  my ($create) = $tester->request_ok(
+    [
+      "Email/set" => {
+        create => {
+          new => {
+            mailboxIds => { $mbox->id => jtrue },
+            htmlBody => [
+              {
+                partId => 'html1',
+                'header:from'  => 'Foo <test@example.org>',
+                'header:X-Foo' => 'x-bar',
+                'header:subject' => 'a test subject',
+                type => 'text/html',
+                cid => 'fooz',
+                language => [ 'US' ],
+              },
+              {
+                partId => 'html2',
+                'header:from'  => 'Foo <test@example.org>',
+                'header:X-Foo' => 'x-bar',
+                'header:subject' => 'a test subject',
+                type => 'text/html',
+                cid => 'fooz',
+                language => [ 'US' ],
+              },
+            ],
+            bodyValues => {
+              text => {
```

Comment:
I would change this to html1 and add another bodyValue for html2 just to make sure we're testing what we think we're testing (since here you have partIds that don't match to bodyValues

Reply:
Aha! I didn't realise that tags `text` were not placeholders. I thought `text` was a keyword. My bad.